Makefile

```diff
@@ -109,9 +120,84 @@ run: generate fmt vet
+	echo "- Creating env files."; \
+	for i in operator forklift rolebinding/{catalog,operator,default}; do \
+		echo "$$DEPLOYMENT_VARS" > hack/deploy/$${i}/deploy.env; \
+	done; \
+	echo "- Creating the operator namespace: $(NAMESPACE)"; \
+	$(KUBECTL) get namespace $(NAMESPACE) &>/dev/null || $(KUBECTL) create namespace $(NAMESPACE); \
+	echo "- Creating the CatalogSource, OperatorGroup and the Subscription manifests"; \
+	$(KUBECTL) apply -k hack/deploy/operator ; \
+	if [ $$OKD -eq 0 ]; then \
+		echo "- Creating the required RoleBindings for the deployment"; \
+		$(KUBECTL) apply -k hack/deploy/rolebinding/default; \
+		$(KUBECTL) apply -k hack/deploy/rolebinding/catalog ; \
+	fi; \
+	echo -n "- Waiting for the catalog pod to exist"; \
+	until $(KUBECTL) -n $(CATALOG_NAMESPACE) get pod -l olm.catalogSource=$(CATALOG_NAME) &>/dev/null; do \
+		sleep 1; echo -n "."; \
+	done; \
+	echo; \
+	if [ $$OKD -eq 0 ]; then \
+	  echo "- Waiting for the catalog pod to be in 'Pending' state"; \
+	  $(KUBECTL) -n $(CATALOG_NAMESPACE) wait --timeout=60s  --for=jsonpath=.status.phase=Pending pod -l olm.catalogSource=$(CATALOG_NAME); \
+	  echo "- Removing the catalog pod"; \
+	  $(KUBECTL) -n $(CATALOG_NAMESPACE) delete pod -l olm.catalogSource=$(CATALOG_NAME); \
+	fi; \
+	echo -n "- Waiting for the operator to be installed"; \
+	until $(KUBECTL) -n $(NAMESPACE) get clusterserviceversion $(OPERATOR_NAME).v$(VERSION) &>/dev/null; do \
+		sleep 1; echo -n "."; \
+	done; \
+	echo; \
+	if [ $$OKD -eq 0 -a "$(CATALOG_NAMESPACE)" != "$(NAMESPACE)" ]; then \
+		echo "- Applying required role bindings"; \
+		$(KUBECTL) apply -k hack/deploy/rolebinding/operator; \
+	fi; \
+	$(KUBECTL) -n $(NAMESPACE)  wait --timeout=60s --for=jsonpath=.status.phase=Succeeded clusterserviceversion $(OPERATOR_NAME).v$(VERSION); \
+	echo "- Creating the Forklift Controller"; \
+	$(KUBECTL) apply -k hack/deploy/forklift; \
+	echo "Done!"
+
+undeploy: kubectl
+	@echo "- Removing the operator namespace: $(NAMESPACE)"
+	@$(KUBECTL) get namespace $(NAMESPACE) -o name 2>/dev/null | xargs -r $(KUBECTL) delete ;
+	@echo "- Removing the CatalogSource"
+	@$(KUBECTL) get catalogsource -n $(CATALOG_NAMESPACE) -o name $(CATALOG_NAME) 2>/dev/null | xargs -r $(KUBECTL) -n $(CATALOG_NAMESPACE) delete;
+	@echo "- Removing the Operator"
+	@$(KUBECTL) get operator $(OPERATOR_NAME).$(NAMESPACE) -o name 2>/dev/null | xargs -r $(KUBECTL) delete;
+	@echo "- Removing the Webhooks"
+	@$(KUBECTL) get mutatingwebhookconfiguration forklift-api -o name 2>/dev/null | xargs -r $(KUBECTL) delete;
+	@$(KUBECTL) get validatingwebhookconfiguration forklift-api -o name 2>/dev/null | xargs -r $(KUBECTL) delete;
+	@echo "- Removing the ConsolePlugin"
+	@$(KUBECTL) get consoleplugin forklift-console-plugin -o name 2>/dev/null | xargs -r $(KUBECTL) delete;
+	@echo "- Removing the OAuthClient"
+	@$(KUBECTL) get oauthclient forklift-ui -o name 2>/dev/null | xargs -r $(KUBECTL) delete;
```

Comment:
this can probably be dropped now that we dropped forklift-ui (the standalone UI)

Reply:
Yeah, I can remove it or leave it so older versions can be fully cleaned up, your choice... 😄